### ISSUE TITLE
The "Add Data to Current View" does not add data on http://viz.neurodata.io

### ISSUE THREAD

**Issue by kkutten1** [neurodata/ndviz#118]
Steps to reproduce....
- Open http://viz.neurodata.io
- Click "+"
- Under token enter "myToken"
- Under channel enter "myChannel"
- Under server select "openconnecto.me"
- Click "Add Data"
- ndviz will respond "Added new layer for channel myChannel".
- Click "Close"

Result: Data is not added despite name of tool
Expected Result: Data is added

Workarounds:
-  Use http://viz.neurodata.io/myToken/myChannel/
- Use a VizProject

**Comment by alexbaden** [comment]
ndviz needs to know a bit about the dataset parameters (xy extent, z slices, etc). Right now that happens before the HTML page is loaded -- the web server grabs the dataset information and sticks it into the webpage, which is then delivered to the client. Reworking that system will be tricky, but also necessary to support a bunch of different functionality (including this issue). It will probably take a few more iterations, especially as I continue to work to make everything more stable.

**Comment by jovo** [comment]
can't @kkutten1 just make a "viz project"?

**Comment by alexbaden** [comment]
Yup. Right now you're using ndviz in "Token View" mode. You just supply a
token and channel(s) and voila -- instance visualization.

VizProjects are useful for combining multiple token/channel combinations,
grouping multiple visualizations as part of a dataview, setting specific
starting colors and/or blending, etc.

Another way to think of it is this -- token view is the power user mode. As
in, I just ingested and want to look at my data, or I'm designing / running
an algorithm and want to overlay the output on top of an existing token /
channel, and don't care about having to click add (because my output token
might change a bunch). VizProjects are for more stable, public/finalized
work.

I started documenting this in the docs. But, I guess I didn't get there or
haven't pushed yet. It would go great here:
http://docs.neurodata.io/ndviz/overview.html#viewing-modes

**Comment by alexbaden** [comment]
#148 needs to happen first
